src/pulse3D/excel_writer.py

```diff
@@ -1029,3 +1064,34 @@ def per_twitch_df(
     df = pd.concat(series_list, axis=1).T
     df.fillna("", inplace=True)
     return df, num_per_twitch_metrics
+
+
+def _get_agg_group_metrics(well_data, well_groups, twitch_widths_range):
+    all_group_metrics = []
+
+    for label, wells in well_groups.items():
+        # group metrics is list of dataframes for well groups
+        group_metrics = [d["metrics"] for d in well_data if d["well_name"] in wells]
+        first_aggregate_well_data = group_metrics[0]
+        combined_df = first_aggregate_well_data[0]
+
+        for group in group_metrics[1:]:
+            combined_df = pd.concat([combined_df, group[0]])
+
+        dfs = init_dfs(twitch_widths_range=twitch_widths_range)
+        aggregate_dfs = dfs["aggregate"]
+
+        for c in combined_df.columns:
+            metric_type = "scalar" if c[0] in CALCULATED_METRICS["scalar"] else "by_width"
```

Comment:
`c` is probably fine here, but `col` is just two more characters and a little more clear